### PR TITLE
21 module for subnets

### LINKED ISSUE #21: modules for subnets

### PR DESCRIPTION
**Pull request by davidebschke**
Here I have create subnets with modules. Many files was deleted and created .

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -14,6 +14,6 @@ terraform {
 
 provider "aws" {
   region = "us-west-2"
+  alias="west2"
```
Comment (fabianschmauder) [inline terraform/main.tf]: do you need this ;)

File: terraform/modules/subnets/variables.tf [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (fabianschmauder) [inline terraform/modules/subnets/variables.tf:1]: the formatting of this file looks a little bit broken. Maybe try to run the formatter

**Review by fabianschmauder — COMMENTED** [review]
LGTM :)

File: terraform/modules/subnets/variables.tf (hunk 2 above)
Comment (davidebschke) [reply terraform/modules/subnets/variables.tf:1]: done

File: terraform/main.tf (hunk 1 above)
Comment (davidebschke) [reply terraform/main.tf]: no I have delete this now

**Pull request merged**